components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -530,6 +530,9 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
             }
 
             userNames = paginateUsers(userNames, limit, offset);
+            //remove duplicated username entries
+            HashSet<String> userNamesSet = new HashSet<String>(Arrays.asList(userNames));
```

Comment:
Explicit type argument can be removed. `new HashSet<>(....`

Reply:
This has been removed in a later PR https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/142/files#diff-4724c8557fb3ee5cb3d1e74b4e78e714L558